Sources/Configuration/Providers/EnvironmentVariables/EnvironmentVariablesProvider.swift

```diff
@@ -142,6 +142,15 @@ public struct EnvironmentVariablesProvider: Sendable {
 
         /// A decoder of arrays from a string.
         var arrayDecoder: EnvironmentValueArrayDecoder
+
+        /// A decoder of bool values from a string
+        static func decodeBool(from string: String) -> Bool? {
+            switch string.lowercased() {
+            case "true", "yes", "1": true
+            case "false", "no", "0": false
+            default: nil
```

Comment:
```suggestion
            default: Bool(string)
```

Let’s use the built-in parser as a fallback.